addons/stock/models/stock_picking.py

```diff
@@ -672,7 +672,7 @@ def _default_picking_type_id(self):
     weight_bulk = fields.Float(
         'Bulk Weight', compute='_compute_bulk_weight', help="Total weight of products which are not in a package.")
     shipping_weight = fields.Float(
-        "Weight for Shipping", compute='_compute_shipping_weight',
+        "Weight for Shipping", compute='_compute_shipping_weight', readonly=False,
```

Comment:
Maybe a warning in stock_delivery it has already been asked to carrier. Since it won't be updated auto